app/cdap/components/SavedRecipeList/index.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { Box } from '@material-ui/core';
+import styled from 'styled-components';
+import { grey } from '@material-ui/core/colors';
+import MyDataPrepApi from 'api/dataprep';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+
+interface IRecipeId {
+  namespace: {
+    name: string;
+    generation: number;
+  };
+  recipeId: string;
+}
+
+export interface IRecipeItem {
+  recipeId: IRecipeId;
+  recipeName: string;
+  description: string;
+  directives: string[];
+  createdTimeMillis: number;
+  updatedTimeMillis: number;
+  recipeStepsCount: number;
+}
+
+interface ISavedRecipeListProps {
+  onRecipeClick: (recipeId: string) => void;
+}
+
+const RecipeNameBox = styled(Box)`
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 150%;
+  color: ${grey[700]};
+  padding: 10px;
+  border-bottom: 1px solid ${grey[300]};
+  cursor: pointer;
+`;
+
+export default function({ onRecipeClick }: ISavedRecipeListProps) {
+  const [recipeList, setRecipeList] = useState([]);
+  const params = {
+    context: getCurrentNamespace(),
+  };
+  const getAllRecipeList = () => {
+    MyDataPrepApi.getRecipeList(params).subscribe((res) => {
+      setRecipeList(res.values);
+    });
+  };
+
+  useEffect(() => {
+    getAllRecipeList();
+  }, []);
+
+  const recipeItemClicked = (recipeItem: IRecipeItem) => {
+    onRecipeClick(recipeItem.recipeId.recipeId);
+  };
+
+  return (
+    <>
+      {recipeList.map((recipeItem: IRecipeItem, recipeIndex) => {
+        return (
+          <RecipeNameBox
```

Comment:
@richadivami 
Box creates a div

>  div: has no semantic meaning, is not keyboard focusable, described as “clickable” by VoiceOver, not listed in VoiceOver’s rotor


Please find more info at https://ux.shopify.com/semantic-html-the-unbearable-rightness-of-being-9b3c493e1791#:~:text=%3Cdiv%3E%3A%20has%20no%20semantic%20meaning%2C%20is%20not%20keyboard%20focusable%2C%20described%20as%20%E2%80%9Cclickable%E2%80%9D%20by%20VoiceOver%2C%20not%20listed%20in%20VoiceOver%E2%80%99s%20rotor

Please find a proper solution for this
if this is not straightforward, please park this for the last, after all other comments are fixed, you can check on this

please plan properly
we need this code perfect at least for us